Explorer/Assets/DCL/Roads/Settings/RoadSettingsAsset.cs

```diff
@@ -14,5 +17,100 @@ public class RoadSettingsAsset : ScriptableObject, IRoadSettingsAsset
 
         IReadOnlyList<RoadDescription> IRoadSettingsAsset.RoadDescriptions => RoadDescriptions;
         IReadOnlyList<AssetReferenceGameObject> IRoadSettingsAsset.RoadAssetsReference => RoadAssetsReference;
+
+#if UNITY_EDITOR
+
+        /// <summary>
+        /// A custom inspector that just shows a search box to make finding road descriptions easier.
+        /// </summary>
+        [CustomEditor(typeof(RoadSettingsAsset))]
+        private class RoadSettingsAssetEditor : Editor
+        {
+            private Vector2Int coordinatesToSearch;
+            private int foundElementIndex = -1;
+            private Vector2Int foundElementCoordinates;
+            private Vector3 foundElementRotation;
+            private string foundElementModel;
+            private bool showNotFoundMessage;
+            private bool showUpdatedMessage;
+
+            public override void OnInspectorGUI()
+            {
+                Color originalColor = GUI.contentColor;
+                GUI.contentColor = Color.cyan;
```

Comment:
Cyan is not legible in light theme.

Reply:
Also, please don't use pure colors. Please look up online some more pleasing tones.

Reply:
Can you please provide an example of a pleasing tone? About the light theme, good point, since we all use the dark tone (AFAIK) I didn't even think about that.

Reply:
I use light theme. In this particular case, I'm not sure what the colour is even trying to accomplish, so I would suggest not using a custom colour at all.

Reply:
As a quick suggestion, take a look at Google's palette while ignoring the second row.
![image](https://github.com/user-attachments/assets/cdf58394-b4f2-4159-8394-7bd60390899a)

Reply:
The intention was to highlight that part of the inspector so it's impossible that anybody misses that search box. I'm removing the coloring code, no problem.